exercises/practice/mazy-mice/test-mazy-mice.bats

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bats
+
+function validate_maze {
+  result="$(\
+    gawk --file mazy-mice.awk --assign Rows=$1 --assign Cols=$2 --assign Seed=$3 \
+    | gawk --file test-maze.awk --assign Rows=$1 --assign Cols=$2 --assign Seed=$3 )"
+  [ "$result" = "The maze is perfect." ]
+}
+
+@test "the smallest square maze is perfect" {
+  #[[ $BATS_RUN_SKIPPED == "true" ]] || skip
+  validate_maze 5 5
+}
+
+@test "the small rectangular maze is perfect" {
+  [[ $BATS_RUN_SKIPPED == "true" ]] || skip
+  validate_maze 5 10
+}
+
+@test "the square maze is perfect" {
+  [[ $BATS_RUN_SKIPPED == "true" ]] || skip
+  validate_maze 10 10
+}
+
+@test "the large rectangular maze is perfect" {
+  [[ $BATS_RUN_SKIPPED == "true" ]] || skip
+  validate_maze 10 20
+}
+
+@test "the rectangular maze with aspect 2:1 is perfect" {
+  [[ $BATS_RUN_SKIPPED == "true" ]] || skip
+  validate_maze 20 10
+}
+
+@test "the huge rectangular maze is perfect" {
+  [[ $BATS_RUN_SKIPPED == "true" ]] || skip
+  validate_maze 20 100
+}
+
+@test "the huge square maze is perfect" {
+  [[ $BATS_RUN_SKIPPED == "true" ]] || skip
+  validate_maze 100 100
+}
+
+@test "if the seed parameter is specified, the perfect maze generated" {
+  [[ $BATS_RUN_SKIPPED == "true" ]] || skip
+  validate_maze 50 50 2342342
+}
+
+@test "if the seed parameter is omitted, random mazes should be generated" {
+  [[ $BATS_RUN_SKIPPED == "true" ]] || skip
+  maze_one=$(gawk --file mazy-mice.awk --assign Rows=8 --assign Cols=16)
+  sleep 1
```

Comment:
sleep shouldn't be needed here ...

Reply:
Oh, nevermind, it's due to the seed.

Reply:
> Oh, nevermind, it's due to the seed.

yes, indeed

Reply:
I suggest adding a comment to avoid future refactoring confusion?

Reply:
I've added the comments to the test.